examples/metrics/plot_sakoe_chiba.py

```diff
@@ -0,0 +1,103 @@
+"""
+================
+Sakoe-Chiba band
+================
+
+This example explains how to set the `window_size` parameter of the Sakoe-Chiba
+band when computing the Dynamic Time Warping (DTW) with
+`method` == "sakoechiba". The Sakoe-Chiba region is defined through a
+`window_size` parameter which determines the largest temporal shift allowed
+from the diagonal in the direction of the longest time series. It is
+implemented in :func:`pyts.metrics.dtw.sakoe_chiba_band`. The window size can
+be either set relatively to the length of the longest time series as a ratio
+between 0 and 1, or manually if an integer is given.
+
+This example visualizes the Sakoe-Chiba band in different scenarios: the
+degenerate case `window_size` = 0, a relative size `window_size` 0.2 and the
+absolute `window_size` = 4. The last two cases are equivalent since
+0.2 * 20 = 4.
+"""
+
+
+# Author: Hicham Janati <hicham.janati@inria.fr>
+#         Johann Faouzi <johann.faouzi@gmail.com>
+# License: BSD-3-Clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+from pyts.metrics import sakoe_chiba_band
+from pyts.metrics.dtw import _check_sakoe_chiba_params
+
+
+######################################################################
```

Comment:
Please add a whitespace so that it doesn't break the code block in the example (see [this example from scikit-learn](https://scikit-learn.org/stable/auto_examples/plot_kernel_ridge_regression.html#sphx-glr-auto-examples-plot-kernel-ridge-regression-py))